ROX:24323: update architecture graphic (#80974)
openshift/openshift-docs: modules/acs-architecture-interaction-between-services.adoc

|Sensor
|⮂
|Scanner

> |Only in {ocp}, can communicate with the lightweight Scanner installed in the secured cluster. This connection allows Sensor to access registries directly from the secured cluster in scenarios where Central might be unable to access them. Scanner communicates with Sensor to request data from `definitions.stackrox.io`.

Reviewer: > |Only in {ocp}, can communicate with the lightweight Scanner installed in the secured cluster.

who can communicate? Sensor?

> Scanner communicates with Sensor to request data from `definitions.stackrox.io`.

Depends on how pedantic we want to be, but Scanner talks to Sensor, in this scenario, to reach out to Central which reaches out to `definitions.stackrox.io`

Reviewer 2: Perhaps this is a bit more complete? 

```suggestion
|Sensor can communicate with the lightweight Scanner installed in the secured cluster. This connection allows Sensor to access registries directly from the secured cluster in scenarios where Central might be unable to access them. Scanner requests updated data from Sensor, Sensor forwards these requests to Central, and Central downloads the requested data from `definitions.stackrox.io`.
```
